test/tcp/TcpServer.cpp

```diff
@@ -62,6 +62,11 @@ class ServerRequestHandler : public RequestHandler {
     response.onNext(folly::IOBuf::copyBuffer("from server2"));
     response.onComplete();
   }
+
+  void handleFireAndForgetRequest(Payload request) override {
+    std::cout << "ServerRequestHandler.handleFireAndForgetRequest "
```

Comment:
I'm going to change the std::cout usages in the handler for now. There are other places in this class that are using cout. I'll talk to Yuri about whether there's a specific reason for that.